src/redis-cli.c

```diff
@@ -6313,15 +6320,15 @@ static int clusterManagerCommandAddNode(int argc, char **argv) {
     redisReply *function_restore_reply = NULL;
     redisReply *function_list_reply = NULL;
     char *ref_ip = NULL, *ip = NULL;
-    int ref_port = 0, port = 0;
-    if (!getClusterHostFromCmdArgs(argc - 1, argv + 1, &ref_ip, &ref_port))
+    int ref_port = 0, ref_bus_port = 0, port = 0;
+    if (!getClusterHostFromCmdArgs(argc - 1, argv + 1, &ref_ip, &ref_port, &ref_bus_port))
         goto invalid_args;
-    if (!getClusterHostFromCmdArgs(1, argv, &ip, &port))
+    if (!getClusterHostFromCmdArgs(1, argv, &ip, &port, NULL))
         goto invalid_args;
     clusterManagerLogInfo(">>> Adding node %s:%d to cluster %s:%d\n", ip, port,
```

Comment:
print ref_bus_port as well?

Reply:
right, thanks, this one we can add, i will go through the code again to see if there's anything else I need to add